ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1393 @@
+{
```

Comment:
<p>The last line says it's 0.9</p><p>It is not a good idea to hardcode numbers like this, since they will change from run to run. You can write that "Модель, представленная в этом ноутбуке, показывает accuracy ~ 90% на тестовой части ...".</p><p>This way you give an overall understanding of models performance while avoiding any hardcode.</p>